applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -1604,6 +1609,8 @@ int le_audio_send(struct encoded_audio enc_audio)
 		return -EINVAL;
 	}
 
+	LOG_DBG("EP %p", (void *)headsets[AUDIO_CH_R].sink_stream.ep);
```

Comment:
It seems a bit random to print the ep pointer for the right headset here, I assume it was used while debugging/developing, but should probably be removed in this PR.

Reply:
oops that was debug printing!